main/cicd/cicd-pipeline/scripts/upload-env-config-if-not-versioned.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -eu
+
+
+cd "$(dirname "${BASH_SOURCE[0]}")/.."
+
+
+aws_profile="${1}"
+deployment_bucket="${2}"
+shift 2
+
+config_s3_path="s3://${deployment_bucket}/settings/"
+
+for environment_name in "$@"; do
+  config_filename="../../config/settings/${environment_name}.yml"
+
+  if [[ -f "${config_filename}" ]]; then
+    git ls-files --error-unmatch "${config_filename}" >/dev/null 2>&1 && (
+      echo "File '${config_filename}' is tracked in version control; not uploaded to S3"
+    ) || (
+      echo "File '${config_filename}' is not tracked in version control"
+      # no need to echo the upload, as the aws command does that anyway
+      # echo "Uploading '${config_filename}' to '${config_s3_path}'"
```

Comment:
nit: can be removed

Reply:
Line 20 cannot be removed.
Line 21 Could, but it is useful to be informative
Line 22-23 can be removed, but they don't hurt explaining why no need to announce what we're about to do (upload).

Reply:
Yeah, comment was meant for just line 23.  Something funky happened when I was adding my comments. Seems like a redundant line to have.